sse2neon.h

```diff
@@ -162,6 +162,10 @@
 #define _MM_FLUSH_ZERO_MASK 0x8000
 #define _MM_FLUSH_ZERO_ON 0x8000
 #define _MM_FLUSH_ZERO_OFF 0x0000
+/* Denormals are zeros mode macros. */
```

Comment:
The comment in GCC looks better. See https://github.com/gcc-mirror/gcc/blob/master/gcc/config/i386/pmmintrin.h#L40

Reply:
However, can we implement a generic [_mm_getcsr](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_getcsr) instead?

Reply:
I am not sure whether GCC comment is better since Clang does not have similar comment and I do not realize why it is an additional bit in MXCSR.

Reply:
_mm_getcsr would be more complicated.
The mapping of the X86 and ARM register should be fully investigated.
At least, it would not be implemented in this pull request.